tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -1151,3 +1219,27 @@ function buildTagToRuns(runTagInfo: {[run: string]: string[]}) {
   }
   return tagToRuns;
 }
+
+// Move the item at sourceIndex to destinationIndex
```

Comment:
I do like the function comment but it should conform to style guide. Also, would be good to mention that you're returning a copy or the array otherwise it implies the move is done on the same array. Maybe:

```
/**
 * Copies headers array with item at sourceIndex moved to destinationIndex.
 */
```

Reply:
I changed it a bit.